firehose-source/README.md

```diff
@@ -0,0 +1,43 @@
+Spring Cloud Stream Firehose Sample
```

Comment:
we should avoid the term "sample" throughout this file - this is simply a source module